### PR TITLE
Added MacOS support on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maddychan**

### CODE REVIEW

**Comment by maddychan** [comment]
Hey Googlebot, I'm a Google employee and accidentally did the commit with my personal email instead of my google.com email - it's on my github profile though, and I switched it to be my primary email now. Thanks!

**Comment by jettisonjoe** [comment]
Fixes #146.

Thanks for doing this! Inline comments to follow.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
```
Comment (jettisonjoe) [inline CONTRIBUTING.md]: I'd put an empty line here to make the .md more readable, even if it actually _renders_ the same either way.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -179,7 +179,36 @@ virtualenv venv
...
 
+Install Xcode and Homebrew Mac. Once that is done:
```
Comment (jettisonjoe) [inline CONTRIBUTING.md]: I would add a very short explanation of what these are and why they're needed, and hyperlink to their respective landing pages.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```
Comment (jettisonjoe) [inline CONTRIBUTING.md]: nit: A space between the # and the comment text makes the comment more readable.
nit+: I like full capitalization and punctuation on comments as if they're sentences.

The [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) doesn't explicitly state these nits, but all its examples are consistent with them.

Comment (jettisonjoe) [inline CONTRIBUTING.md]: I would just call these "dependencies". I think that's more clear as OpenHTF is on what could be called the "dependent" side of the relation here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -179,7 +179,36 @@ virtualenv venv
...
+
+#Change to the openhtf directory 
+cd openhtf
+
+#Create a new virtualenv
+virtualenv venv
+
+#Activate the new virtualenv
+. venv/bin/activate
+```
```
Comment (jettisonjoe) [inline CONTRIBUTING.md]: Should add a `python setup.py develop` step at the end.

File: CONTRIBUTING.md (hunk 2 above)
Comment (maddychan) [inline CONTRIBUTING.md]: It seems that a gcc compiler is needed for Homebrew and pip, and xcode is one way to provide that but not the only way. What do you suggest I include? Just Homebrew and the landing page?

Comment (jettisonjoe) [inline CONTRIBUTING.md]: Ah, I didn't realize xcode was where we were getting pip from, too. I think it's reasonable to recommend and link to both, as long as we provide a few words explaining why.

**Comment by maddychan** [comment]
Just redid the steps with my newly reimaged mac and it went perfectly. yay!

**Comment by grybmadsci** [comment]
LGTM, looks awesome, thanks!  next up is windows ;) (don't really, it's a nightmare, we'll deal with it later...)

**Comment by jettisonjoe** [comment]
Awesome, merging now!

**Pull request merged**